maintenance/wikia/PortabilityMetric/updateMainPageFilterInMetricProcessor.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @ingroup Maintenance
+ *
+ * @desc For current wiki, find an ID of it's Main Page
```

Comment:
added a comment about cleaning `mainpagefilter_b` column before running the script